### PR TITLE
Add local node build status to network modification tree

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/components/graph/nodes/network-modification-node.js [hunk 1]
```diff
@@ -165,7 +204,7 @@ const NetworkModificationNode = (props) => {
                         : classes.networkModification
                 }
             >
-                <div className={getClassForBanner(props.data.buildStatus)}>
+                <div className={getClassForBanner(props.data.buildStatusLocal)}>
                     {props.data.buildStatus === 'BUILDING' && (
```
Comment (flomillot) [inline src/components/graph/nodes/network-modification-node.js:208]: buildStatusComputed, the progress bar will never display

**Review by flomillot — CHANGES_REQUESTED** [review]

**Comment by flomillot** [comment]
Tests OK (need to fix progress circle)

File: src/components/graph/nodes/network-modification-node.js (hunk 1 above)
Comment (Meklo) [reply src/components/graph/nodes/network-modification-node.js]: Indeed, I just fixed this

File: src/components/network-modification-tree-pane.js [hunk 2]
```diff
@@ -360,7 +360,8 @@ export const NetworkModificationTreePane = ({
                     createTreeNode(studyUuid, element.id, insertMode, {
                         name: response,
                         type: type,
-                        buildStatus: BUILD_STATUS.NOT_BUILT,
+                        buildStatusLocal: BUILD_STATUS.NOT_BUILT,
+                        buildStatusLocalComputed: BUILD_STATUS.NOT_BUILT,
```
[the remark this answers is not in the document]
Comment (Meklo) [reply src/components/network-modification-tree-pane.js]: Fixed

**Review by flomillot — APPROVED** [review]

**Review by Walid-Sahnoun — CHANGES_REQUESTED** [review]
in this example, when i cut the node N6 (under N5) and paste it under N4
the node N6 doesn't change it own status
image

**Comment by Meklo** [comment]
> in this example, when i cut the node N6 (under N5) and paste it under N4 the node N6 doesn't change it own status image

Noted, I fixed it

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
test ok

**Review by SlimaneAmar — APPROVED** [review]
Perharps need to add a status build enum constant
In an another US

**Pull request merged**